### DISCUSSION TITLE
Dask on HPC (compute nodes has no internet access)

### DISCUSSION THREAD

**Discussion by iuryt** [dask/distributed#6782] [Q&A]
Hi all,

I am currently using a cluster that I can use their portal to open Jupyterlab from the browser, but the compute nodes has no access to the internet. I would like to use Dask and monitor Dask Dashboard. Anyone has any advice or tutorial for a case like this? I am getting errors by running with SLURM jobqueue, but I would like to check tutorials and examples before creating an issue.

**Comment by jacobtomlinson** [comment]
You shouldn't need internet access to use the dashboard. If you can access Jupyter Lab then you could use the Dask extension to view the dashboard. It uses a proxy inside Jupyter to make the dashboard pages available to you.

https://github.com/dask/dask-labextension
